### PR TITLE
Transparency ThirdParty Accountant Use Case

### PR DESCRIPTION
**Pull request by MikeMcGarry**

### CODE REVIEW

File: examples/use-cases/lusid_sample_data.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    is a 1 in 36**4 = 1,679,616 chance of a collision. 
+    
+    Input
+    num_scopes: Integer value for the number of identifiers that we need
+    
+    Output
+    scopes: List of scope identifiers
+    """
+    
+    scopes = []
```
Comment (rizwansaeed) [inline examples/use-cases/lusid_sample_data.py]: Use list comprehension

Comment (MikeMcGarry) [reply examples/use-cases/lusid_sample_data.py]: Resolved.

File: examples/use-cases/lusid_sample_data.py [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+    for portfolio_group in portfolios['portfolio_group_name'].unique():
+        client_portfolios[portfolio_group] = []
+                
+        for portfolio in portfolios.loc[portfolios['portfolio_group_name'] == portfolio_group, 'portfolio_name']:
+            client_portfolios[portfolio_group].append(portfolio)
+    
+    return client_portfolios
+
+
+def fetch_instrument_universe(csv_file, paper=False):
```
[the remark this answers is not in the document]
Comment (MikeMcGarry) [reply examples/use-cases/lusid_sample_data.py]: Added for clarification

File: examples/use-cases/lusid_sample_data.py [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    Input
+    csv_file: The name of the csv file including the extension i.e. .csv
+    
+    Output
+    client_portfolios: Dictionary containing the client portfolios
+    """
+    
+    portfolios = import_file(csv_file)
+                
+    client_portfolios = {}
```
[the remark this answers is not in the document]
Comment (MikeMcGarry) [reply examples/use-cases/lusid_sample_data.py]: Not a dictionary comprehension but using dict.setdefault we can make this much nicer.

**Pull request merged**